example-deployment/docker/reporting-server/reporting-server-base.dockerfile

```diff
@@ -11,6 +11,8 @@ RUN . /opt/rmf/setup.bash && \
 
 FROM rmf-web/builder
 
+COPY --from=0 /root/rmf-web/packages/reporting-server/ /root/reporting-server/
```

Comment:
Doesn't this supersedes the next line,
`COPY --from=0 /root/rmf-web/packages/reporting-server/dist/ .`

Reply:
No, it's copying to another folder, this is used only for migration purposes.